tensorbay/sensor/intrinsics.py

```diff
@@ -415,6 +578,39 @@ def loads(cls: Type[_T], contents: Dict[str, Dict[str, float]]) -> _T:
             A :class:`CameraIntrinsics` instance containing information from
             the contents dict.
 
+        Example:
+            >>> contents = {
+                "cameraMatrix": {
```

Comment:
...     "cameraMatrix": {

There should be ellipses at the front of this line, also should the following lines.
The indentation is incorrect.